.gitignore

```diff
@@ -16,3 +16,4 @@ test/tmp
 test/version_tmp
 tmp
 gemfiles/*gemfile.lock
+.byebug_history
```

Comment:
I see no reason to add byebug to the gem, even as a development dependency. Can you remove the related parts?

Reply:
@fschwahn it's more convenient than puts debugging.